### PR TITLE
Fix error_log complaint about clueboard/66_hotswap/gen1 layout macro

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

```
"Error: clueboard/66_hotswap/gen1: LAYOUT: Number of elements in info.json does not match! info.json:70 != LAYOUT:68"
```
Now matches `LAYOUT` as shown here:
https://clueboard.co/support/qmk-configurator#clueboard_66_hotswap_gen1

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
Travis error is unrelated.

**Comment by fauxpark** [comment]
Just noticed there is an info.json one level higher which is functionally identical to what I've done here, so I'm deleting the `gen1` info.json and cleaning up that one instead.

File: keyboards/clueboard/66_hotswap/info.json [hunk 1]
```diff
@@ -1,25 +1,165 @@
...
-        {"label": "LSHIFT", "x": 0, "y": 3, "w": 2.25}, {"label": "Z", "x": 2.25, "y": 3}, {"label": "X", "x": 3.25, "y": 3}, {"label": "C", "x": 4.25, "y": 3}, {"label": "V", "x": 5.25, "y": 3}, {"label": "B", "x": 6.25, "y": 3}, {"label": "N", "x": 7.25, "y": 3}, {"label": "M", "x": 8.25, "y": 3}, {"label": "COMMA", "x": 9.25, "y": 3}, {"label": "PERIOD", "x": 10.25, "y": 3}, {"label": "SLASH", "x": 11.25, "y": 3}, {"label": "RSHIFT", "x": 12.25, "y": 3, "w": 2.25}, {"label": "UP", "x": 14.5, "y": 3},
-        {"label": "LCTRL", "x": 0, "y": 4, "w": 1.25}, {"label": "LGUI", "x": 1.25, "y": 4, "w": 1.25}, {"label": "LALT", "x": 2.5, "y": 4, "w": 1.25}, {"label": "SPACE", "x": 3.75, "y": 4, "w": 6.25}, {"label": "RALT", "x": 10, "y": 4, "w": 1.25}, {"label": "RGUI", "x": 11.25, "y": 4}, {"label": "FN", "x": 12.25, "y": 4, "w": 1.25}, {"label": "LEFT", "x": 13.5, "y": 4}, {"label": "DOWN", "x": 14.5, "y": 4}, {"label": "RIGHT", "x": 15.5, "y": 4}
-      ]
+    "keyboard_name": "Clueboard 66% HotSwap",
+    "maintainer": "skullydazed",
+    "width": 16.5,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/clueboard/66_hotswap/info.json:7]:
```suggestion
        "LAYOUT_all": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6, "y": 0},
                {"x": 7, "y": 0},
                {"x": 8, "y": 0},
                {"x": 9, "y": 0},
                {"x": 10, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0, "w": 2},

                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1, "w": 1.5},
                {"x": 1.5, "y": 1},
                {"x": 2.5, "y": 1},
                {"x": 3.5, "y": 1},
                {"x": 4.5, "y": 1},
                {"x": 5.5, "y": 1},
                {"x": 6.5, "y": 1},
                {"x": 7.5, "y": 1},
                {"x": 8.5, "y": 1},
                {"x": 9.5, "y": 1},
                {"x": 10.5, "y": 1},
                {"x": 11.5, "y": 1},
                {"x": 12.5, "y": 1},
                {"x": 13.5, "y": 1, "w": 1.5},

                {"x": 15.5, "y": 1},

                {"x": 0, "y": 2, "w": 1.75},
                {"x": 1.75, "y": 2},
                {"x": 2.75, "y": 2},
                {"x": 3.75, "y": 2},
                {"x": 4.75, "y": 2},
                {"x": 5.75, "y": 2},
                {"x": 6.75, "y": 2},
                {"x": 7.75, "y": 2},
                {"x": 8.75, "y": 2},
                {"x": 9.75, "y": 2},
                {"x": 10.75, "y": 2},
                {"x": 11.75, "y": 2},
                {"x": 12.75, "y": 2, "w": 2.25},

                {"x": 0, "y": 3, "w": 1.25},
                {"x": 1.25, "y": 3},
                {"x": 2.25, "y": 3},
                {"x": 3.25, "y": 3},
                {"x": 4.25, "y": 3},
                {"x": 5.25, "y": 3},
                {"x": 6.25, "y": 3},
                {"x": 7.25, "y": 3},
                {"x": 8.25, "y": 3},
                {"x": 9.25, "y": 3},
                {"x": 10.25, "y": 3},
                {"x": 11.25, "y": 3},
                {"x": 12.25, "y": 3},
                {"x": 13.25, "y": 3, "w": 1.25},

                {"x": 14.5, "y": 3},

                {"x": 0, "y": 4, "w": 1.25},
                {"x": 1.25, "y": 4, "w": 1.25},
                {"x": 2.5, "y": 4, "w": 1.25},
                {"x": 3.75, "y": 4, "w": 2.75},
                {"x": 6.5, "y": 4, "w": 2.25},
                {"x": 8.75, "y": 4, "w": 1.25},
                {"x": 10, "y": 4, "w": 1.25},
                {"x": 11.25, "y": 4},
                {"x": 12.25, "y": 4, "w": 1.25},
                {"x": 13.5, "y": 4},
                {"x": 14.5, "y": 4},
                {"x": 15.5, "y": 4}
            ]
        },
        "LAYOUT": {
```

Comment (fauxpark) [reply keyboards/clueboard/66_hotswap/info.json:7]: `gen1` doesn't contain a `LAYOUT_all` though...

Comment (noroadsleft) [reply keyboards/clueboard/66_hotswap/info.json]: IIRC the API searches the keyboard.h file for the layout macros, then searches info.json for each macro's layout data. But no matter, separate files works.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**